docs/3.0rc/concepts/artifacts.mdx

```diff
@@ -3,32 +3,27 @@ title: Artifacts
 description: Prefect artifacts are persisted outputs designed for human consumption and available in the UI.
 ---
 
-Artifacts are persisted outputs such as tables, Markdown, or links.
-They are stored on Prefect Cloud or a Prefect server instance and rendered in the Prefect UI.
-Artifacts make it easy to track and monitor the objects that your flows produce and update over time.
+Artifacts are visually rich annotations on Prefect objects. They are visual metadata intended to be produced in flows or tasks but consumed by humans. They come in standardized formats such as tables, progress idicators, images, Markdown, or links. They are stored on Prefect Cloud or a Prefect server instance and rendered in the Prefect UI.
+Artifacts make it easy to visualize outputs or side effects that your flows produce and capture updates over time.
 
 ![Markdown artifact sales report screenshot](/3.0rc/img/ui/md-artifact-info.png)
 
-Published artifacts may be associated with a particular task run or flow run.
-Artifacts can also be created outside of any flow run context.
-
-Whether you're publishing links, Markdown, or tables, artifacts provide a powerful and flexible way to showcase data within your workflows.
-
-With artifacts, you can easily manage and share information with your team, providing valuable insights and context.
+Artifacts are associated with a particular Prefect object, usually a task run or a flow run.
```

Comment:
Can they be associated with anything else?

Reply:
We don't want them to to global or unassociated anymore -- maybe there's a better way to say that?

Reply:
I would just say "Artifacts are associated with a task or flow run."